app/_landing_pages/insomnia.yaml

```diff
@@ -144,45 +129,33 @@ rows:
               blocks:
                 - type: text
                   text: | 
-                    See the full [feature comparison &rarr;](/insomnia/feature-comparison)
```

Comment:
We should also delete the feature comparison page

Reply:
done